Code/Framework/AzToolsFramework/AzToolsFramework/Editor/RichTextHighlighter.cpp

```diff
@@ -0,0 +1,50 @@
+#include "RichTextHighlighter.h"
+
+namespace AzToolsFramework
+{
+    RichTextHighlighter::RichTextHighlighter(QString displayString)
+        : m_displayString(displayString){};
```

Comment:
nit: I'd run clang-format on this new file (`{}` should be on a new line and we don't want the trailing `;`)